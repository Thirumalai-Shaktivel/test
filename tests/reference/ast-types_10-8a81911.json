{
    "basename": "ast-types_10-8a81911",
    "cmd": "lfortran --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/types_10.f90",
    "infile_hash": "2d1357260f2fc09e94884cf4955b12f3f35407dc0b74358b32019f36",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-types_10-8a81911.stdout",
    "stdout_hash": "35d4ad0887e1f0540e513f02b408d75b75c777badc53af5ec39c65c8",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}