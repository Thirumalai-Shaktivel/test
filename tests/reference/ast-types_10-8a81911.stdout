(TranslationUnit [(Program types_10 [] [(ImplicitNone [])] [(Declaration (AttrType TypeInteger [] () None) [] [(i [] [] () None () ()) (j [] [] () None () ())]) (Declaration () [(SimpleAttribute AttrSave)] [(i [] [] () None () ())]) (Declaration () [(SimpleAttribute AttrSave)] [(j [] [] () None () ())])] [] [(Subroutine f [(i)] [] () [] [] [] [(Declaration (AttrType TypeInteger [] () None) [(AttrIntent In)] [(i [] [] () None () ())]) (Declaration (AttrType TypeInteger [] () None) [(SimpleAttribute AttrSave)] [(j [] [] () None () ())])] [(= 0 j i)] []) (Subroutine g [(i)] [] () [] [] [] [(Declaration (AttrType TypeInteger [] () None) [(AttrIntent In)] [(i [] [] () None () ())]) (Declaration (AttrType TypeInteger [] () None) [] [(j [] [] () None () ())]) (Declaration () [(SimpleAttribute AttrSave)] [(j [] [] () None () ())])] [(= 0 j i)] []) (Subroutine h [(i)] [] () [] [] [] [(Declaration (AttrType TypeInteger [] () None) [(AttrIntent In)] [(i [] [] () None () ())]) (Declaration (AttrType TypeInteger [] () None) [] [(j [] [] () None () ())]) (Declaration () [(SimpleAttribute AttrSave)] [])] [(= 0 j i)] [])])])
